packages/opentelemetry-instrumentation/README.md

```diff
@@ -0,0 +1,153 @@
+# OpenTelemetry Instrumentation for web and node
+
+[![Gitter chat][gitter-image]][gitter-url]
+[![NPM Published Version][npm-img]][npm-url]
+[![dependencies][dependencies-image]][dependencies-url]
+[![devDependencies][devDependencies-image]][devDependencies-url]
+[![Apache License][license-image]][license-image]
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation
+```
+
+## Usage in Node
+
+```typescript
```

Comment:
This example is very complex and hard to follow. Nothing seems wrong, but we should consider creating a guide that explains each component.

Reply:
I would be in favor of adding that once I finish the graphql plugin as basically new plugin for graphql is helping me to refactor the plugins in general too - if you know what I mean. It should not change much but some small changes might yet appear.

Reply:
simplified